pizza_cutter/des_pizza_cutter/_constants.py

```diff
@@ -8,17 +8,13 @@
 # kernel extends outside bounds in resampling
 BMASK_RESAMPLE_BOUNDS = 2**26
 
-# this bit marks masked areas around gaia stars. Note this is different from
-# the STAR bit that is already set for masked areas around saturated stars
-
-BMASK_GAIA_STAR = 2**25
```

Comment:
I'd argue that we want to keep this flag defined here. I know it is funny, but it makes sense to keep it with the rest of the flags.

Reply:
OK